modules/plugins/hydra/hydra.nix

```
@@ -0,0 +1,8 @@
+{lib, ...}: let
+  #inherit (lib) types;
+  inherit (lib.options) mkEnableOption;
+in {
+  options.vim.hydra = {
+    enable = mkEnableOption "Creating custom submodes and menus [nvimtools/hydra.nvim]";
+  };
```

```suggestion
  options.vim.utility.hydra = {
    enable = mkEnableOption "utility for creating custom submodes and menus [nvimtools/hydra.nvim]";
  };
```

It's better to have plugins under specific categories instead polluting top-level `vim.*`. You'll also want to move this under plugins/utility.

---

I just assumed hydra was a broader spectrum plugin. It's fine in the utility folder too.